packages/components/src/components/amount-input/amount-input.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import Input from '../input';
+
+type TAmountInput = {
+    currency: string;
+    decimalPoints?: number;
+    disabled?: boolean;
+    initialValue?: number;
+    label?: React.ReactNode;
+    maxDigits: number;
```

Comment:
```suggestion
    maxDigits?: number;
```
@rostislav-deriv Let's make this one optional too and give it a default value 🙏🏻 

_The less required prop we have makes the easier it gets for devs who are not familiar with the component and want to use it for the first time as they will just render it and then start playing around the props._